src/assessments/adaptable-content/test-steps/text-spacing.tsx

```diff
@@ -16,43 +18,23 @@ const textSpacingDescription: JSX.Element = (
 
 const textSpacingHowToTest: JSX.Element = (
     <div>
-        This test uses the{' '}
-        <NewTabLink href="https://www.html5accessibility.com/tests/tsbookmarklet.html">
-            Text spacing
-        </NewTabLink>{' '}
-        bookmarklet to adjust text spacing in the target page.
-        <ol>
+        The visual helper for this requirement adjusts the target page's text styling as follows:
+        <ul>
             <li>
-                Open{' '}
-                <NewTabLink href="https://www.html5accessibility.com/tests/tsbookmarklet.html">
-                    Text spacing
-                </NewTabLink>{' '}
-                in a new browser window.
+                <Markup.Term>Letter spacing</Markup.Term> (tracking) at 0.12 times the font size{' '}
```

Comment:
nitpick: The `{' '}`s aren't necessary here. We use those when we need to preserve spaces in between lines that JSX processing would normally remove whitespace for. For example, in something like

```jsx
<>
    some
    <em>emphasized</em>
    text
</>
```

...JSX's whitespace-collapsing behavior would normally result in there being no spaces between the three words in the rendered HTML. `{' '}` is the usual trick to avoid that for inline elements where you want to preserve the space, but it's generally fine/desirable to omit the space at the ends of `display: block` elements (like the `</li>` end tags in this code).

Some StackOverflow threads with more thorough descriptions:
* [How to avoid JSX component from condensing when React.js rendering it?](https://stackoverflow.com/q/32686728/2825668)
* [Best practice when adding whitespace in JSX](https://stackoverflow.com/q/40264084/2825668)

Reply:
oh awesome! good to know!